SpywareFilter/sections/specific.txt

```diff
@@ -292,6 +292,7 @@ android-tools.ru###text-8
 ||bursadabugun.com/haber/imp-
 ||i.jsrdn.com/i/1.gif?r=
 ||api.impresa.pt/trackercookie
+/Scripts/ticimax/analytics.js$domain=elleshoes.com
```

Comment:
@wout-codes 
let's change `/Scripts/ticimax/analytics.js$domain=bataryapil.com|journey.com.tr|tozlu.com` to `/analytics.js$domain=bataryapil.com|journey.com.tr|tozlu.com` and add elleshoes.com to it